ci/tests/Jenkinsfile.e2e-prs

```diff
@@ -101,7 +103,7 @@ pipeline {
                 -m \"${params.TEST_MARKERS}\" \
                 -k \"${params.KEYWORD_EXPRESSION}\" \
                 --apk=${params.APK_NAME} \
-                --build=PR-${params.PR_ID} \
+                --build=PR-${params.PR_ID}-${nowDate} \
```

Comment:
The [`utils`](https://github.com/status-im/status-jenkins-lib/blob/master/vars/utils.groovy) library in [status-jenkins-lib](https://github.com/status-im/status-jenkins-lib) already has a [`timestamp()`](https://github.com/status-im/status-jenkins-lib/blob/eadd4e962e35109931645cd90b05bf8bab92d830/vars/utils.groovy#L15-L19) function.

You should be able to just add this at the top of the file:
```groovy
library 'status-jenkins-lib@v1.4.0'
```
And then be able to use `utils.timestamp()`.